integration

```diff
@@ -1596,6 +1596,7 @@
   "toringer/home-assistant-metnowcast",
   "toringer/home-assistant-sildre",
   "toringer/home-assistant-thermostat-setback",
+  "Toxo666/ha_froeling_modbus",
```

Comment:
Since the README is written entirely in German and this integration is for Fröling heating systems (primarily available in German-speaking countries), you need to add country restrictions to your `hacs.json` file.

Please update your `hacs.json` to include:
```json
{
  ...
  "country": ["DE", "AT"],
  ...
}
```

If Fröling S3200 systems are available in additional countries beyond Germany and Austria, please include those ISO country codes as well.

After making this change, please create a new GitHub release with the updated configuration, then mark this PR as ready for review.

**Reference:** https://www.hacs.xyz/docs/publish/start/#country